sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -40,7 +55,17 @@ def iterable():
 def error():
     return Response(stream_json_error(), status=400)
 
+
+@streams_api.route('/string', methods=['GET'])
+def string():
+    return Response(
+        streaming_test(), status=200, mimetype="text/plain"
+    )
```

Comment:
It doesn't look like this code gets used in this PR - seems we can safely remove it.